pkg/controller/certificates/requestmanager/requestmanager_controller.go

```diff
@@ -69,6 +72,11 @@ type controller struct {
 	recorder                 record.EventRecorder
 	clock                    clock.Clock
 	copiedAnnotationPrefixes []string
+
+	// fieldManager is the string which will be used as the Field Manager on
+	// fields created or edited by the cert-manager Kubernetes client during
+	// Apply API calls.
+	fieldManager string
```

Comment:
Suggestion: remove this as the 'apply' functionality that would use this field manager is removed in a later commit

Reply:
I've realised that we should probably be adding the field manager to `Create` call. Though it would be the same anyway because of the user agent, it is better to be more explicit. I'll update CertificateRequest Create calls, and open another PR for Certificates.

Reply:
We also don't need to gate setting the field manager on `Create` API calls, since it is not Apply.

Reply:
> We also don't need to gate setting the field manager on Create API calls, since it is not Apply.

Ok, that makes sense. So the only time when another actor would modify fields being created by this field manager is when annotations are being applied and in that case A) if server-side feature gate is enabled, the update would be forced  like before B) if server-side feature gate is not enabled, the update does not care about who was the previous field manager like before

Reply:
I guess we could have also added the new field managers to all `UpdateOptions` too?